ambari-server-spi/src/main/java/org/apache/ambari/server/AmbariException.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ambari.server;
+
+import java.io.IOException;
+
+@SuppressWarnings("serial")
+public class AmbariException extends IOException {
```

Comment:
I am surprised that this does not cause ambiguity issues.   What if this or the AmbariException class in the `ambari-server` project changes?

I assume fixing this issue is part of the "future commits" noted in the description of this PR.

Reply:
I'm getting rid of `AmbariException` in `ambari-server`. Let me do that as part of this commit.